docs/react-v9/contributing/component-implementation-guide.md

```diff
@@ -0,0 +1,264 @@
+
+### Spec Review
+
+The reviewers will help to double check the spec, and the API is consistent with other Fluent UI controls. The review is _not_ intended to check every aspect of the design to ensure that the control works for all use cases--you are still accountable to create a high quality design for the control.
+
+To get a review started:
+
+- Create a pull request that contains `SPEC.md` and `.types.ts` (if applicable).
+  - If you have any open questions, call them out with a comment on the PR.
+  - Assign all of the core spec reviewers. Your PR should get signed off by at least one of these reviewers to be considered complete:
+    - Levi (levithomason)
+    - Miro (miroslavstastny)
+    - Ben (behowell)
+  - Anyone can provide input or feedback on Specs
+- If needed, create a spec review meeting or use Spec Review - Office Hours to walk through the spec, and include a link to the PR in the meeting invite.
+  - If you have access to the Fluent UI Internal Teams channel, post a link to the spec in the Convergence channel so people know to look at the spec. At-mention the reviewers to ensure it's on their radar.
+- The spec does not have to be final with all issues or open questions resolved before merging the first draft.
+  - If the open questions are large enough to warrant their own discussion, consider creating separate issues to track those open questions.
+
+# Phase 2: Initial Implementation
+
+Once the first draft of the spec has been reviewed, you can get started on the implementation.
+
+## Implementation
+
+**Requirements**
+
+- Use React hooks
+- Use makeStyles
+- Respect Figma tokens (and use the Provider)
+- Respect API principles, shorthands and slots handling
+- No dependency on v0/v7/v8
+- Add tsdoc `/** */` comments to all public APIs in your component
+
+You can follow examples of other converged components for the structure of the implementation, such as:
+
+- [Button](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-button/src/components/Button)
+- [Avatar](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-avatar/src/components/Avatar)
+
+## Documentation
+
+### Storybook stories
+
+The primary public documentation for our controls is in the form of storybook stories demonstrating the various features and recommended usage of the control. The stories are in the /stories subfolder of the component package. In general, each story should cover only one feature of a control, and follow best practices for using the control. The `yarn create-component` command creates a single story for each component. If you're building a larger, more complex, compositional component, it may not make sense to publish a story for every component, so some may just be slightly opinionated flavors of existing v9 components. Be sure to be intentional about the story you're telling around the main component experience.
```

Comment:
>  The stories are in the /stories subfolder of the component package.

Note: this will change in upcoming weeks
- https://github.com/microsoft/fluentui/issues/30516
- https://github.com/microsoft/fluentui/pull/30514/files#diff-fd1f65124cb8a5f905a337bf23e6dd67b607808793d84f21178c25c80b3a8028R283

Reply:
Roger that!